exercises/practice/binary-search-tree/binary-search-tree.jl

```diff
@@ -0,0 +1,2 @@
+# Create a (Mutable) Struct BinarySearchTree, which has fields: data, left, right
+# Also write a sort method, which returns a sorted array of the elements in a BinarySearchTree
```

Comment:
```suggestion
# Also write a sort method, which returns a sorted array of the elements in a BinarySearchTree.

```

It seems that this should be sentences in a comment, but surely the EOL is important as well for the last line of this stub.